homework-submission/homework-week3/react-hyf/server/server.js

```diff
@@ -0,0 +1,10 @@
+const express = require('express');
+const app = express();
+
+app.use(express.static('./../react-hyf-client/build'));
+
+app.get('/', function(req, res) {
+  res.sendFile('./../react-hyf-client/build/index.html');
```

Comment:
This is not what exactly meant by using the proxy in your application, what is happening here is just you are opening your application to be available on two different ports which what any server can serve it as.

The imporant part of using a proxy in your application is just to cheat _(if is it possible to say)_ client side by telling it:
> you don't need to contact any other third party services to get your data, but instead you may send the request to '/hello/world' and everything going to be handled.

And what will happen here is just re-routing the request from port let's say `localhost:3000` to `localhost:1234` or from `google.com` as client to `apis.google.com` as an api server, or may be other third party service doesn't related to our application at all.

All of that the Browser will think he is just requesting from the same site and won't guess it's been re-routed. 😃